static/css/main.css

```diff
@@ -1,182 +1,182 @@
 body a {
-    color: #2294e0;
+  color: #2294e0;
```

Comment:
this file is auto-formatted via `make format-web`

Reply:
Hmm, I am a little worried that prettier has different versions/config? It looks like this just changed whitespace?

Reply:
oh actually looks good. 2 is the default.